plugins/com.google.cloud.tools.eclipse.sdk.test/src/com/google/cloud/tools/eclipse/sdk/internal/CloudSdkUpdateJobTest.java

```diff
@@ -98,12 +98,11 @@ public void testFailureSeverity()
     when(managedCloudSdk.isInstalled()).thenReturn(false);
 
     CloudSdkUpdateJob job = newCloudSdkUpdateJob();
-    job.setFailureSeverity(IStatus.WARNING);
     job.schedule();
     job.join();
 
     IStatus result = job.getResult();
-    assertEquals(IStatus.WARNING, result.getSeverity());
+    assertEquals(IStatus.ERROR, result.getSeverity());
```

Comment:
Looking at the name of this test, I think it was specifically for testing the severity is `WARNING` when calling `setFailureSeverity(WARNING)`.

Reply:
removed